src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -3944,7 +3944,26 @@ void ExportMusicXml::rest(Rest* rest, staff_idx_t staff)
     if (b && ExportMusicXml::canWrite(b)) {
         notations.tag(_xml, b);
         articulations.tag(_xml);
-        _xml.tag(b->isCaesura() ? "caesura" : "breath-mark");
+        if (b->isCaesura()) {
+            _xml.tag("caesura");
+        } else {
+            QString breathMarkType;
+            switch (b->symId()) {
+            case SymId::breathMarkTick:
+                breathMarkType = "tick";
+                break;
+            case SymId::breathMarkUpbow:
+                breathMarkType = "upbow";
+                break;
+            case SymId::breathMarkSalzedo:
+                breathMarkType = "salzedo";
+                break;
+            default:
+                breathMarkType = "comma";
+            }
+
+            _xml.tag("breath-mark", breathMarkType);
+        }
```

Comment:
Woukldn't this code be needed for chors too, like at https://github.com/musescore/MuseScore/blob/cbfa4d4be68a843136ee229106c0c85414e0a3f8/src/importexport/musicxml/internal/musicxml/exportxml.cpp#L2993 ?

Reply:
Oh, I wasn't aware of chords being handled separately (first time looking at the code). I'll check it out!

Reply:
Indeed somewhat confusing.

Reply:
Probably needs the text file adjusted too

Reply:
> testNoteAttributes2_ref.xml needs to get adjusted too then,

Yep, I noticed that also. I extracted the breath mark handling into a separate function in my latest commit.

Reply:
Looks good to me

Reply:
You may want to check whether my integration of this into 3.x is correct, see #9000

Reply:
> Looks good to me.
> 
> You may want to check whether my backport of this for 3.x is correct, see #9000.

Yep, that looks good to me, thanks for backporting!